xml/System.Xml/XmlReaderSettings.xml

```diff
@@ -123,7 +123,9 @@ The following example creates an <xref:System.Xml.XmlReader> that uses an <xref:
 |--------------|-------------------|
 |<xref:System.Xml.XmlReaderSettings.Async%2A>|`false`.|
 |<xref:System.Xml.XmlReaderSettings.CheckCharacters%2A>|`true`.|
+|<xref:System.Xml.XmlReaderSettings.CloseInput%2A>|`false`.|
```

Comment:
I looked at the online docs, and this property seems to exist even for older versions of .NET Framework: https://learn.microsoft.com/en-us/dotnet/api/system.xml.xmlreadersettings.closeinput?view=netframework-3.0. 
I don't actually have a way to verify this, though.

As for the value, I used the default value defined in .NET source code: https://github.com/dotnet/runtime/blob/b4a1fa2293f31aa694e5a433aca676c29b50de81/src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReaderSettings.cs#L419